tests/sos_tests.py

```diff
@@ -819,7 +820,7 @@ def setUp(self):
         # seems awkward, but check_installed() and remove() are not exposed
         # together with install_distro_packages()
         self.installer = software_manager
-        self.sm = self.installer.SoftwareManager()
+        self.sm = self.installer.manager.SoftwareManager()
```

Comment:
With the changes between `SoftwareManager()` and `distro_packages` that we've found, we can un-awkward this bit and clean it up, e.g. simply:

```python
self.sm = software_manager.manager.SoftwareManager()
```

And drop the notes on lines 819+